docs/DeveloperGuide.adoc

```diff
@@ -77,6 +77,32 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+=== Use case: Rename tag
+
+*MSS*
+
+. User requests to rename tags
+. AddressBook shows the list of tags
+. User a selects a specific tag in the list, e.g. 'friend'
```

Comment:
No need to provide an example here. The given example was only for your understanding.
Your use case should tackle a general problem, here it is renaming a tag.